Gemfile

```diff
@@ -3,8 +3,12 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '2.5.1'
 
+# shrine
+
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '~> 5.2.1'
+
+gem 'pry'
```

Comment:
pry should be only on deb and test environments

Reply:
Maybe I didn't explain correctly, you should move the `pry` gem to the test and development environments, and remove `byebug`, its not required if you're using pry